README.md

```diff
@@ -166,11 +171,24 @@ To help you in finding projects suitable for you, this awesome list uses the fol
 * [🖼][p_win][📊][a_xl] [VBA-IDE-Code-Export](https://github.com/spences10/VBA-IDE-Code-Export) - Addin contains a code importer and exporter for use with git (or any VCS).
 * [🖼][p_win][📊][a_xl][✒️][a_wd][🔒][o_pass][🏺][o_32] - AndyPope's Visual Ribbon Editor.
 * [🖼][p_win][📊][a_xl] [Custom UI XML Editor](https://yoursumbuddy.com/ribbon-customui-xml-editor/) - Addin for directly adding, editing and validating ribbon XML (Excel 2010+).
+* [🖼][p_win][⭐][a_all] [MZ-Tools](https://www.mztools.com/) - VBA Development Tool
 * [🖼][p_win][⭐][a_all] [VbPeg](https://github.com/wqweto/VbPeg) - A parser generator for VBA. Converts PEG grammar like [this](https://github.com/wqweto/VbPeg/blob/master/test/Runner/peg/Kscope/grammar.peg) into [VBA code like this](https://github.com/wqweto/VbPeg/blob/master/test/Runner/peg/Kscope/cKscope.cls). Very useful if your implementing a new programming language in VBA.
 * [🖼][p_win][⭐][a_all] [VBA Resource File Editor](http://leandroascierto.com/blog/vba-resource-file-editor/) - Store other files inside your excel/word/powerpoint files for later use with this handy tool. 
 
+## Miscellaneous
+
+* [👑][p_all][⭐][a_all] [Excel Name Manager](https://jkp-ads.com/excel-name-manager.asp) - A treeview control replacement by JKP and Peter Thornton coded entirely in VBA.
+* [👑][p_all][⭐][a_all] [Excel Flex Find](https://jkp-ads.com/excel-flexfind.asp) - A treeview control replacement by JKP and Peter Thornton coded entirely in VBA.
```

Comment:
Again I assume description is a mistake as you've included treeview already above. I feel like FlexFind is an example (as it is an example of a VBA application). Feel like this can be approached in 1 of 2 ways. On the one hand the flexfind custom grid control looks neat. So it's a good example of a UI. On the other hand, it's just a generally useful Excel Tool. Would be happy to have a `Examples > Excel tools` section

## Actions

* MoveTo `Examples > UI Userforms` or `Examples > Excel tools` - your choice.
* Change description